### PR TITLE
Corrections in certificates docs

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/2460

@wjun, does this cover everything from internal bug 2339700? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/vch_cert_reqs.md [hunk 1]
```diff
@@ -128,17 +126,16 @@ For information about how to automatically generate a client certificate during
 
 ### Custom Client Certificate <a id="clientcert"></a>
 
-For the VCH to trust the CA that you use to sign the client certificate, the CA must include the following elements:
+For the VCH to trust the CA that you use to sign the client certificate, the client certificate must include the following elements:
```
Comment (wjun) [inline docs/user_doc/vic_vsphere_admin/vch_cert_reqs.md]: Please remove the first part and only keep "The client certificate must include the following elements:"

File: docs/user_doc/vic_vsphere_admin/vch_cert_reqs.md [hunk 2]
```diff
@@ -128,17 +126,16 @@ For information about how to automatically generate a client certificate during
...
 
 - The name or address of the system from which the Docker client accesses the server in the subject or subject alternative name. This can be an FQDN or a wildcard domain.
```
Comment (wjun) [inline docs/user_doc/vic_vsphere_admin/vch_cert_reqs.md]: Remove this requirement that actually made the user confusion in the bug.

Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/vch_cert_reqs.md]: Fixed in https://github.com/vmware/vic-product/pull/2513/commits/1509674bf0a4a9f5606d17d7f14db4424ebbf49d.

File: docs/user_doc/vic_vsphere_admin/vch_cert_reqs.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/vch_cert_reqs.md]: Fixed in https://github.com/vmware/vic-product/pull/2513/commits/1509674bf0a4a9f5606d17d7f14db4424ebbf49d.

**Comment by stuclem** [comment]
Thanks @wjun !

**Pull request merged**